### PR TITLE
Add runbook

### PR DESCRIPTION
**Pull request by bboykov**
# Description

## What

Add runbook for the service.

## Why

[JIRA ticket](https://financialtimes.atlassian.net/browse/UPPSF-1181).

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [ ] Feature
- [X] Documentation
- [ ] Breaking change
- [X] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

File: runbooks/runbook.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+The Public Organisations API is a micro-service which aims to provide Organisation and related data given an Organisation identifier.
+
+## Code
+
+public-org-api
+
+## Primary URL
+
+<https://github.com/Financial-Times/public-organisations-api>
```
Comment (karsov) [inline runbooks/runbook.md:11]: This is a public API, its URL is:
```suggestion
<http://api.ft.com/organisations>
```

File: runbooks/runbook.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+FullyAutomated
+
+## Failback Process Type
+
+PartiallyAutomated
+
+## Failover Details
+
+See the [failover guides for more details](https://github.com/Financial-Times/upp-docs/tree/master/failover-guides)
```
Comment (karsov) [inline runbooks/runbook.md:66]: I would prefer to have here the cluster specific failover guide:
```suggestion
The service is deployed in both Delivery clusters. The failover guide for the cluster is located here: https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster
```

File: runbooks/runbook.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+## Key Management Details
+
+There is no key rotation procedure for this system.
+
+## Monitoring
+
+Look for the pods in the cluster health endpoint and click to see pod health and checks:
+
+- <https://upp-prod-delivery-eu.upp.ft.com/__health>
+- <https://upp-prod-delivery-us.upp.ft.com/__health>
```
Comment (karsov) [inline runbooks/runbook.md:101]: The specific links would be way more useful:
```suggestion
- Delivery-Prod-EU health: <https://upp-prod-delivery-eu.upp.ft.com/__health/__pods-health?service-name=public-organisations-api>
- Delivery-Prod-US health: <https://upp-prod-delivery-us.upp.ft.com/__health/__pods-health?service-name=public-organisations-api>
```

**Review by karsov — CHANGES_REQUESTED** [review]
The dependency on public-concepts-api is missing.

File: runbooks/runbook.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+
+No
+
+## Contains Sensitive Data
+
+No
+
+## Dependencies
+
+- concepts-rw-neo4j
```
Comment (karsov) [inline runbooks/runbook.md:54]:
```suggestion
- public-concepts-api
```

**Review by karsov — DISMISSED** [review]

File: runbooks/runbook.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+content
+
+## Known About By
+
+- marina.chompalova
+- elitsa.pavlova
+- ivan.nikolov
+- miroslav.gatsanoga
+
```
Comment (ekaneva) [inline runbooks/runbook.md:35]: Kalin is missing from the list. :)

**Review by ekaneva — CHANGES_REQUESTED** [review]
Whoever takes care of this moving forward should combine all commits in one.

**Review by karsov — APPROVED** [review]

**Review by ekaneva — DISMISSED** [review]
Please just combine all these commits in just one as now they do not bring any additional information - adding a runbook is sufficient.

**Review by karsov — DISMISSED** [review]

**Comment by MiroslavGatsanoga** [comment]
Looks good to go, just rebased onto master.

**Review by karsov — DISMISSED** [review]

File: runbooks/runbook.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+FullyAutomated
+
+## Failback Process Type
+
+PartiallyAutomated
+
+## Failover Details
+
+The service is deployed in both Delivery clusters. The failover guide for the cluster is located here: https://github.com/Financial-Times/upp-docs/tree/master/failover-guides/delivery-cluster
```
Comment (ekaneva) [inline runbooks/runbook.md:71]: Missing `<` `>` around the URL.

File: runbooks/runbook.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+
+A public API that gives basic information about Things in the knowledge graph database.
+
+## Code
+
+public-org-api
+
+## Primary URL
+
+<http://api.ft.com/organisations>
```
Comment (ekaneva) [inline runbooks/runbook.md:11]: I thought that we're using URLs in this format for the runbook:
<https://upp-prod-delivery-glb.upp.ft.com/__public-organisations-api/>
not the public API URLs.

The idea is to point the person looking into the runbook directly to the service.

**Review by ekaneva — CHANGES_REQUESTED** [review]

File: runbooks/runbook.md (hunk 7 above)
Comment (MiroslavGatsanoga) [reply runbooks/runbook.md:11]: I think we should use the public endpoints for public APIs, the double underscore endpoints should be for services that are not public.

Comment (mitagg) [reply runbooks/runbook.md:11]: I second Miro's comment, it makes more sense that way.

Comment (MiroslavGatsanoga) [reply runbooks/runbook.md:11]: Noted.
FYI we are not introducing it here for the first time:
https://github.com/Financial-Times/public-brands-api/blob/master/runbooks/runbook.md
https://github.com/Financial-Times/public-people-api/blob/master/runbooks/runbook.md
https://github.com/Financial-Times/public-things-api/blob/master/runbooks/runbook.md
https://github.com/Financial-Times/public-suggestions-api/blob/master/runbooks/runbook.md

Comment (ekaneva) [reply runbooks/runbook.md:11]: Yes, my point was that the runbooks are mixed. I've also sent the `people-api`. My sample just had 1 Public API example and 3 non-public ones. :) So, I am ok with using the public one.

**Review by karsov — APPROVED** [review]

**Pull request merged**